examples/07_data_observatory/discovery.ipynb

```diff
@@ -0,0 +1,253 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Discoverying content of interest in the Data Observatory\n",
+    "\n",
+    "CARTO's Data Observatory is a powerful tool for exploring the available datasets in our data lake. Through the discovery API you would be able to navigate through the datasets and their properties, thus knowing in advanced which sources may be of interest for you before even requesting access to them. "
```

Comment:
I'd write this as:

```suggestion
    "The Discovery API is a powerful tool for exploring the available datasets in our Data Observatory catalog. Through the Discovery API you would be able to navigate through the datasets and their variables, thus knowing in advanced which sources may be of interest for you before even requesting access to them. "
```

Reply:
Done! I changed all the mentions in the notebook.